test/unit/helpers-test.js

```diff
@@ -16,16 +16,5 @@ describe('helpers', function() {
         assert.equal(tester[0], 'hello');
         assert.equal(tester[1], 'hi');
     });
-
-    it('log will not thow if console is cleared', function() {
-        var tmpConsole = window.console;
-
-        window.console = null;
-        // this should not break
-        utils.log('testing');
-
-        // restore actual window console
-        window.console = tmpConsole;
-    });
```

Comment:
Removing tests because they log to the console is not necessary. What's the big deal?